bedrock/firefox/templates/firefox/all.html

```diff
@@ -115,6 +115,19 @@
   </main>
 {% endblock %}
 
+{% block email_form %}
+  <aside id="newsletter-subscribe" class="ga-section" data-ga-label="Keep up with all things Firefox.">
+    <div class="content container">
+      {% if l10n_has_tag('fxnewsletter_subscribe_update') %}
```

Comment:
We don't translate this page. Only Japanese does, but it's untracked by our systems.